packages/app/src/app/overmind/namespaces/editor/state.ts

```diff
@@ -56,7 +59,7 @@ type State = {
   mainModule: Derive<State, Module>;
   currentPackageJSON: Derive<State, Module>;
   currentPackageJSONCode: Derive<State, string>;
-  parsedConfigurations: Derive<State, any>;
+  parsedConfigurations: Derive<State, ParsedConfigurationFiles> | null;
```

Comment:
Is it possible for them to be null?

Reply:
https://github.com/codesandbox/codesandbox-client/blob/1aca6b669358bc42a3487f2b0e3b5cb33c8ca5be/packages/app/src/app/overmind/namespaces/editor/state.ts#L145-L146

Reply:
Hm, but at the same time, `currentSandbox: Derive<State, Sandbox>` doesn't have ` | null`. What's your opinion on this @christianalfoni? Should we make this `| null`?

The tricky thing is that when we're in `/s/:id` we can assume that `currentSandbox` is set, and thus that `parsedConfigurations` is true. A perfect solution would be to make this nullable and say that for a specific path/actions we can assume that the value is set, but that seems impossible to do.

Reply:
I don't think that's possible to type tbh.

In our code we can force it to be set using the bang operator (`!`) in specific parts or the code tho.

Reply:
Even if we are on `/s/:id` the Sandbox needs to be downloaded, so it is not necessarily there :)

Reply:
So I can just leave the type nullable right?

Reply:
Jup!